sechub-webui/src/main/resources/templates/login.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:th="https://www.thymeleaf.org">
+<head>
+<title>Please Log In</title>
+<link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+	<h1>Please Log In</h1>
+	<div th:if="${param.error}">Invalid username and password.</div>
+	<div th:if="${param.logout}">You have been logged out.</div>
+	<form th:action="@{/login}" method="post">
+		<div>
+			<input type="text" name="username" placeholder="Username" />
```

Comment:
Can you please add csrf protection

ala


```
<input 
  type="hidden" 
  th:name="${_csrf.parameterName}" 
  th:value="${_csrf.token}" />
```
see https://www.baeldung.com/csrf-thymeleaf-with-spring-security

I would like to have this security mechanism from the beginning.